### ISSUE TITLE
Client-side error when closing window with GridDragSource

### ISSUE THREAD

**Issue by ToArnld** [vaadin/framework#9101]
Vaadin version: 8.1.0.alpha5

I have a `Window` with two `Grid` extended by `GridDragSource`.

When closing the `Window` I get the following javascript-exception:
```javascript
AppWidgetset-0.js:6163 Tue Apr 18 17:03:54 GMT+200 2017 com.vaadin.client.ApplicationConfiguration
SEVERE: (TypeError) : Cannot read property 'yb' of undefined
com.google.gwt.core.client.JavaScriptException: (TypeError) : Cannot read property 'yb' of undefined
	at Unknown.Vgb(AppWidgetset-0.js)
	at Unknown.hSb(AppWidgetset-0.js)
	at Unknown.HSb(AppWidgetset-0.js)
	at Unknown.Kvb(AppWidgetset-0.js)
	at Unknown.lIb(AppWidgetset-0.js)
	at Unknown.eIb(AppWidgetset-0.js)
	at Unknown.LHb(AppWidgetset-0.js)
	at Unknown.MHb(AppWidgetset-0.js)
	at Unknown.rFb(AppWidgetset-0.js)
	at Unknown.eval(AppWidgetset-0.js)
	at Unknown.Rh(AppWidgetset-0.js)
	at Unknown.Uh(AppWidgetset-0.js)
	at Unknown.eval(AppWidgetset-0.js)
	at Unknown._f(http://localhost:8080/Project/VAADIN/vaadinPush.debug.js?v=8.1.0.alpha5)
	at Unknown._invokeFunction(http://localhost:8080/Project/VAADIN/vaadinPush.debug.js?v=8.1.0.alpha5)
	at Unknown._invokeCallback(http://localhost:8080/Project/VAADIN/vaadinPush.debug.js?v=8.1.0.alpha5)
	at Unknown._websocket.onmessage(http://localhost:8080/Project/VAADIN/vaadinPush.debug.js?v=8.1.0.alpha5)
```
After that the loading indicator flashes and the app needs to be reloaded.

Everything works fine without Drag'n'Drop functionality. 
When I extend the `Grid` with the `GridDragSource` the error occurs when closing the window.:
```java
GridDragSource<Task> dragSource = new GridDragSource<>(grid); 
```

**Comment by ptishaton** [comment]
Hi,
Same here using Vaadin 8.1.0.alpha6. A Grid using DnD feature inside a Tabsheet trigger this exception when the user will be changing tab...

**Comment by pleku** [comment]
Hi @ptishaton , that issue was caused by a recent change and will be fixed with #9341 that will be included in next release (alpha9 or beta1).

The issue in this ticket is not affected by the #9341 fix.

**Comment by pleku** [comment]
Ok now the improved fix in #9341 will resolve this issue too.
